toc.md

```diff
@@ -740,10 +742,12 @@ module Workerholic
 end
 ```
 
-When workerholic starts, it'll load the app which load rails if it detects a specific file in Rails, and then we require our own active job adapter and load that in along with the rest of the rails application.
+When Workerholic starts, it'll load the app which load Rails if it detects that it's working a Rails application, and then we require our own active job adapter and load that along with the rest of the Rails application.
```

Comment:
This sentence is a little strange IMO, I'd prefer:

> When Workerholic starts, it loads the main application in order to have access to the job classes so that it can perform the jobs on the processing side. In case a Rails app is detected it will load the Rails application along with our own active job adapter.